resume-buddy/src/main/java/com/google/sps/servlets/ReviewerDataServlet.java

```diff
@@ -43,7 +43,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
   /**
    * @return the request parameter, or the default value if the parameter was not specified by the
-   * client
+   *     client
```

Comment:
Nit: weird spacing before "client"

Reply:
I think this is Shreya's code, maybe auto format did that then came with my commit.